crates/docs/src/snippet.rs

```diff
@@ -59,6 +59,8 @@ pub struct SnippetConfig {
     pub package_name: Option<String>,
     #[serde(default)]
     pub ignored_output: bool,
+    #[serde(default)]
+    pub not_replace_network: bool,
```

Comment:
Please change the name so there's no negation, it makes the code harder to understand.